package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-constructor",
-  "version": "0.6.0",
+  "version": "0.6.1",
```

Comment:
version is upto `0.7.0` as I speak 👀

Reply:
type rather

Reply:
Thanks will update.  I'm expecting merge conflicts with other pending tickets ahead of this one too so may need to make other changes.